src/sentry/api/endpoints/accept_project_transfer.py

```diff
@@ -126,7 +126,9 @@ def post(self, request: Request) -> Response:
 
             if not is_team_org_owner:
                 return Response({"detail": "Invalid team"}, status=400)
+            from sentry.utils import metrics
```

Comment:
Generally we prefer to import things at the top of the file, unless there's some circular import problem we need to work around